packages/messaging/src/models/ContentCard.ts

```diff
@@ -0,0 +1,47 @@
+/*
+    Copyright 2024 Adobe. All rights reserved.
+    This file is licensed to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law
+    or agreed to in writing, software distributed under the License is
+    distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS OF
+    ANY KIND, either express or implied. See the License for the specific
+    language governing permissions and limitations under the License.
+*/
+
+import { PersonalizationSchema } from './PersonalizationSchema';
+
+type ContentCardTemplate = 'SmallImage';
+type DismissButtonStyle = 'circle' | 'none' | 'simple';
+
+export interface ContentCard {
```

Comment:
Where do we use this interface?
Is this for UI?  
Can we update the sample app to show the user how to implement with the UI?

Reply:
We have a different UI project that may come later for them. ContentCard is part of a union type for Proposition Item so they can get it from the `getProposition` call

Reply:
We think it is still good to show how to retrieve this in the sample app in a log window.  Possible to implement this in the sample app?

And any documentation should be added for this ability for the release?